start-cron.js

```diff
@@ -0,0 +1,19 @@
+const config = require('./config')
+const log = require('./app/services/log')
+var CronJob = require('cron').CronJob
+var generateDirectPayments = require('./app/services/data/insert-task-generate-direct-payments')
```

Comment:
prefer insertTaskGenerateDirectPayments, so it is clear that this does not actually generate direct payments, just insert the task